docs/index.html

```diff
@@ -0,0 +1,191 @@
+            font-size: 40px;
+            margin: 40px 0 20px 0;
+        }
+
+        .banner p {
+            margin: 20px 0 40px;
+        }
+
+        .column {
+            max-width: 960px;
+            margin-left: auto;
+            margin-right: auto;
+            padding-left: 20px;
+            padding-right: 20px;
+        }
+
+        .button {
+            font-size: 24px;
+            background: #cc7a6f;
+            color: #fafafa;
+            border-radius: 4px;
+            padding: 8px 16px;
+            box-shadow: 1px 3px 3px rgba(0, 0, 0, 0.3);
+            text-shadow: 0 1px 3px rgba(0, 0, 0, 0.3);
+            text-decoration: none;
+            margin: 30px;
+        }
+
+        .container {
+            display: flex;
+            min-height: 500px;
+        }
+
+        article {
+            margin: 20px;
+            padding: 20px;
+            color: #666;
+            flex: 1 1 0
+        }
+
+        article p {
+            margin: 20px 0;
+        }
+
+        footer {
+            text-align: right;
+            padding: 20px;
+        }
+    </style>
+</head>
+<body>
+<header>
+    <nav class="column">
+        <h1><span class="logo">&lt;ნ&gt;</span> Nakadi</h1>
+        <ul>
+            <li><a href="manual.html">Docs</a></li>
+            <li><a href="manual.html#nakadi-event-bus-api">API</a></li>
+            <li><a href="https://github.com/zalando-nakadi/">Community</a></li>
+        </ul>
+        <ul class="right">
+            <li>
+                <a href="https://github.com/zalando/nakadi">GitHub</a>
+            </li>
+        </ul>
+    </nav>
+</header>
+
+<section class="banner">
+    <div class="column">
+        <h1>Nakadi</h1>
+        <h3>Event Bus Broker</h3>
+        <p>A distributed event bus that implements a RESTful API abstraction instead of Kafka-like queues</p>
+        <a class="button" href="manual.html">Get Started</a>
+    </div>
+</section>
+<section class="column container">
+
+    <article><h1>RESTfull</h1>
+        <p>Abstract event delivery via a secured RESTful API. This allows microservices teams to maintain service
+            boundaries, and not directly depend on any specific message broker technology. Access to the API can be
+            managed and secured using OAuth scopes.</p>
+    </article>
+    <article><h1>JSON Schema</h1>
+        <p>Enable convenient development of event-driven applications and asynchronous microservices. Event types can be
+            defined with schemas and managed via a registry. Nakadi also has optional support for events describing
+            business processes and data changes using standard primitives for identity, timestamps, event types, and
+            causality.</p>
+    </article>
+    <article><h1>Flexibility</h1>
+        <p>Nakadi instances are stateless. They can be run on AWS with auto-scaling.</p>
+        <p>Using Timelines it is easy to move the traffic to another cluster without moving the data and any service
```

Comment:
`Using Timelines it is easy` -> `Using Timelines, it is easy`
`without moving the data and any service degradation` -> `without moving the data, and without service interruption`.

Also, this is the first mention of timelines, and this does not explain what they are.